scipy/stats/morestats.py

```diff
@@ -952,13 +952,29 @@ def boxcox(x, lmbda=None, alpha=None):
         Input array.  Must be positive 1-dimensional.  Must not be constant.
     lmbda : {None, scalar}, optional
         If `lmbda` is not None, do the transformation for that value.
-
         If `lmbda` is None, find the lambda that maximizes the log-likelihood
         function and return it as the second output argument.
     alpha : {None, float}, optional
         If ``alpha`` is not None, return the ``100 * (1-alpha)%`` confidence
         interval for `lmbda` as the third output argument.
         Must be between 0.0 and 1.0.
+    optimizer : callable, optional
+        If `lmbda` is None, `optimizer` is the scalar optimizer used to find
```

Comment:
From someone who does not really know about the method, `lmbda` is not defined as a param and this might not be clear as of how it works.

Reply:
Yes, it should not be written with single-backticks (and thus rendered in italics) because it is not a parameter of this function. Also, the statement "If `lmbda is None..." is not neededhere; it was erroneously copied from `boxcox`.

However, `lmbda`, is defined above:

![image](https://user-images.githubusercontent.com/6570539/114433640-e7a92400-9b76-11eb-93c9-a38f9b066c24.png)

What do you think about changing it to double-backticks here to match the definition above?

Reply:
OK sure if out of scope we can do a follow up PR to fix it.